mix/ClientModel.cpp

```diff
@@ -149,60 +169,66 @@ void ClientModel::executeSequence(std::vector<TransactionSettings> const& _seque
 		try
 		{
 			bytes contractCode = compilerRes->bytes();
-			std::vector<dev::bytes> transactonData;
-			QFunctionDefinition* f = nullptr;
-			ContractCallDataEncoder c;
-			//encode data for all transactions
+			m_client->resetState(_balance);
+			onStateReset();
 			for (auto const& t: _sequence)
```

Comment:
If I read `t.functionId` in line 217 below and want to know what is actually happening, I have to go to this line to discover that the type of `t` might be derived from the type of `_sequence` which is caught by value in this lambda and defined as a `vector` of `TransactionSetting` in line 155.
I know that I might be one of the few developers that like explicit types, but please choose at least a descriptive name for `t`. :-)